main/thirdparty/org/openintents/intents/FileManagerIntents.java

```diff
@@ -1,130 +1,130 @@
-/* 
- * Copyright (C) 2008 OpenIntents.org
- *

```

Comment:
Sorry, overseen yesterday.
I think we should let this file unchanged.
The current source is at https://github.com/openintents/filemanager/blob/master/FileManager/src/main/java/org/openintents/intents/FileManagerIntents.java
Instead we should update this file with the current implementation.